.github/scripts/build_app_windows.bat

```diff
@@ -7,12 +7,11 @@ rem See https://github.com/dlemmermann/JPackageScriptFX
 rem The script depends on various environment variables to exist in order to
 rem run properly:
 rem
-rem PROJECT_VERSION=1.1.0-SNAPSHOT
-rem APP_VERSION=1.0.0
+rem APP_VERSION=%APP_VERSION%
```

Comment:
This sets the APP_VERSION value to that of APP_VERSION?
Why? this line does not make sense to me...

Reply:
That may well be ;) but the comment is still confusing in saying X=%X%